src/components/Switch/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,356 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
```

Comment:
Why ? 🤔

Reply:
This error appears in each story on `e.target.checked`, not sure why

Reply:
You need to type your onChange :)